Library/Homebrew/test/test_keg.rb

```diff
@@ -304,4 +304,35 @@ def test_removes_broken_symlinks_that_conflict_with_directories
     keg.unlink
     keg.uninstall
   end
+
+  def test_mach_o_files_skips_hardlinks
+    a = (HOMEBREW_CELLAR/"a")/"1.0"
```

Comment:
Ruby 1.8.7 can be a real pain. I'm so looking forward to #404 and the freedom this will grant us. (Hopefully, Ruby 1.8.7 becomes a distant memory very soon.)